### ISSUE TITLE
Some question about the embedding, loss function and data.

### ISSUE THREAD

**Issue by hanzhongyuan** [arvind385801/paraphraseGen#6]
I am very interested in your work and want to follow your research. I read the code and have some questions. I would be very grateful if you can get your advice.

1. The code needs two embeddings, one for the original sentences and the other for the generated sentences, why? If we use the same embedding, how will it affect the performance?
2. In the line 156 in the rvae.py,  you set loss = 79 * cross_entropy + kld_coef(i) * kld.  And for kld_coef(i), the function defined two constants, 3500 and 1000. What factors are taken into account to set these parameters?  
3. Are the training document 'train.txt' and the test document 'test.txt' the same to the documents used in the paper (50K train data and 4K test data)? Can you give me 100K and 150K corpus?

**Comment by hanzhongyuan** [comment]
I am very interested in your work and want to follow your research. I read the code and have some questions. I would be very grateful if you can get your advice.

1. The code needs two embeddings, one for the original sentences and the other for the generated sentences, why? If we use the same embedding, how will it affect the performance?
2. In the line 156 in the rvae.py,  you set loss = 79 * cross_entropy + kld_coef(i) * kld.  And for kld_coef(i), the function defined two constants, 3500 and 1000. What factors are taken into account to set these parameters?  
3. Are the training document 'train.txt' and the test document 'test.txt' the same to the documents used in the paper (50K train data and 4K test data)? Can you give me 100K and 150K corpus?
